.github/workflows/doctests.yml

```diff
@@ -4,9 +4,16 @@ on:
   push:
     branches:
       - doctest*
-  repository_dispatch:
-  schedule:
-    - cron: "0 0 * * *"
+  pull_request:
+    paths:
+      - "src/**"
+      - "tests/**"
+      - ".github/**"
+      - "templates/**"
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
No I don't think we want to run those on every PR since it's rather slow. Once per day was the target IIRC @LysandreJik

Reply:
I originally modified only `documentation_tests.txt`, and as @patrickvonplaten suggested, I merged his PR  #15010 to see what will happen (which includes this part). I try to observe this process to get a better idea about how doctest is done here.

The last check gives (https://github.com/huggingface/transformers/runs/4736720693?check_suite_focus=true)

```
   Error response from daemon: OCI runtime create failed: container_linux.go:367: starting container process caused: process_linux.go:495: container init caused: Running hook #0:: error running hook: exit status 1, stdout: , stderr: nvidia-container-cli: initialization error: nvml error: driver/library version mismatch: unknown
  Error: failed to start containers: 0c2e0a498ded2a80298395260946cbd8781323ba2abd5270c7e40e2294f477c1
  Error: Docker start fail with exit code 1
```

I think what @patrickvonplaten suggested is to see which doc examples will fail, but here Docker failed to start - I am not sure what to do in this case.

Reply:
100% agree that they should only be run once per day. Just think it's a good idea to run the tests in this PR to be sure the PR correctly runs the tests. Before merging this PR this file should be reverted, but I'd leave it for now as it is.